### PR TITLE
2020-10-16 status update

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: _posts/2020-10-16-status-update.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "2020-10-16: IAC Team Status Update"
+author: DavidS
+categories:
+  - team
+  - status
+tags:
```
Comment (sanfrancrisko) [inline _posts/2020-10-16-status-update.md:8]: We'll need to fill this out

File: _posts/2020-10-16-status-update.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-docker#666`][puppetlabs-docker-pr-666]: Thanks to [optiz0r][optiz0r]
```
Comment (sanfrancrisko) [inline _posts/2020-10-16-status-update.md:15]: @david22swan can you fill out the thank you messages below?

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]
Just comments on the blog post itself.

The other changes to the generation scripts LGTM 👍

File: _posts/2020-10-16-status-update.md (hunk 1 above)
Comment (DavidS) [reply _posts/2020-10-16-status-update.md:8]: thanks for reminding me!

Comment (DavidS) [reply _posts/2020-10-16-status-update.md:8]: done

File: _posts/2020-10-16-status-update.md (hunk 2 above)
Comment (DavidS) [reply _posts/2020-10-16-status-update.md]: I've added the PR titles now. anything specific you'd like to see?

**Pull request merged**